<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>25</y>
      <width>570</width>
      <height>400</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>メイリオ</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Outer Tube</string>
    </property>
    <widget class="QSlider" name="verticalSliderOuter">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>50</width>
       <height>350</height>
      </rect>
     </property>
     <property name="maximum">
      <number>270</number>
     </property>
     <property name="value">
      <number>135</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderOuter">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>340</y>
       <width>350</width>
       <height>50</height>
      </rect>
     </property>
     <property name="maximum">
      <number>270</number>
     </property>
     <property name="value">
      <number>135</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ovvvl">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Vertical</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="ovhvl">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>210</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Horizontal</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxVO">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>240</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> deg</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>270.000000000000000</double>
     </property>
     <property name="value">
      <double>135.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxHO">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>260</y>
       <width>240</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> deg</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>270.000000000000000</double>
     </property>
     <property name="value">
      <double>135.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>25</y>
      <width>650</width>
      <height>961</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>メイリオ</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Camera / Detection</string>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>280</y>
       <width>600</width>
       <height>600</height>
      </rect>
     </property>
     <property name="backgroundBrush">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </widget>
    <widget class="QComboBox" name="cameraComboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>320</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelResolution">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>250</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Resolution  1920 x 1080</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelAspect">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>250</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Aspect Ratio  16 : 9</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="cuptureButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>151</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Capture</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="flipCheckBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>61</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Flip</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="dbboxDispCheckBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>341</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Display Darkness Detected Box</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="labelDiff">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>501</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Difference from the center : </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelControll">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>205</y>
       <width>501</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Controll : </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="applyButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>220</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Applying</string>
     </property>
    </widget>
    <widget class="QComboBox" name="detectorComboBox">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>120</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>570</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>メイリオ</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Arduino</string>
    </property>
    <widget class="QLabel" name="arduinoLogLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>541</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Port : COMX , BaudRate : 115200 
 255,255,255,255,255,255,255,255,255</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="recordButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Record</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
